src/hooks.ts

```diff
@@ -839,3 +840,17 @@ export function useRaycastVehicle(
   }, deps)
   return [ref, api]
 }
+
+type MaterialFn = () => MaterialOptions
+
+export function useMaterial(fn?: MaterialFn): Material {
+  const [material] = useState(() => new Material(fn && fn()))
```

Comment:
We are not using state to hold onto physics entities in the hooks as they live inside of the worker.